test/src/test/java/hudson/PluginSEC925Test.java

```diff
@@ -11,13 +11,15 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.concurrent.Future;
+import org.junit.Ignore;
 
 //TODO merge it within PluginTest after the security release
 public class PluginSEC925Test {
     
     @Rule
     public JenkinsRule r = new JenkinsRule();
     
+    @Ignore("TODO observed to fail in CI with 404")
```

Comment:
Some sort of network outage, it seems.

Reply:
This one seems unnecessary, haven't seen it fail a substantial number of times.